### PR TITLE
Add official packages section and add Solus

### PR DESCRIPTION
**Pull request by Jacalz**
Added a section for unofficial packages in to the installing brave documentation.

## Test Plan:
- verified it looked good through GitHub rendering the page

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

### CODE REVIEW

File: docs/source/installing-brave.rst [hunk 1]
```diff
@@ -243,3 +244,18 @@ CentOS/RHEL
...
+Unofficial packages
+============================
+
+.. highlight:: console
+
+Solus 
+-----------
+::
+
+    sudo eopkg it brave
```
Comment (srirambv) [inline docs/source/installing-brave.rst]: This will be only for release channel I suppose?

Comment (Jacalz) [reply docs/source/installing-brave.rst]: Yeah that is correct

Comment (srirambv) [reply docs/source/installing-brave.rst]: May be good to make it `Unofficial packages (Release Channel only)` in a follow up PR.

Comment (Jacalz) [reply docs/source/installing-brave.rst]: I will see what I can do and follow up with a new PR in a few days 👍

**Pull request merged**